.github/workflows/build.yml

```diff
@@ -0,0 +1,86 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main, develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.1.0
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3.6.0
+        with:
+          distribution: adopt
+          java-version: 11
+      - name: Cache Gradle and wrapper
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
```

Comment:
Can't we use our own `.github/actions/gradle-cache` action?

Reply:
Updated: [570cb52](https://github.com/GetStream/stream-log/pull/11/commits/570cb5284601c8c10105b000d33a08b7fc762c32)